src/MSGEQ7.h

```diff
@@ -104,6 +111,10 @@ class CMSGEQ7{
 		// little hack to take as many arguments as possible
 		// to execute several functions for the analogPins
 	}
+
+	// This calculates the value based on what MSGEQ7_data_t is defined.
+	// This assumes the analog pin resolution is 10 bit.
```

Comment:
I am not sure if this comment is correct.

Reply:
> I am not sure if this comment is correct.

That comment is for the method getAnalogReadPin. Depending on what Bit value is set, it returns the MSGEQ7_data_t value.